src/main/java/cotato/backend/api/weather/WeatherController.java

```diff
@@ -0,0 +1,37 @@
+package cotato.backend.api.weather;
+
+import cotato.backend.domain.weather.dto.HourlyWeatherResponseDto;
+import cotato.backend.domain.weather.dto.WeatherDetailResponseDto;
+import cotato.backend.domain.weather.service.query.WeatherQueryService;
+import cotato.backend.global.common.ApiResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/locations/{id}/weather")
+@RequiredArgsConstructor
+public class WeatherController {
+
+    private final WeatherQueryService weatherQueryService;
+
+    @GetMapping
+    public ApiResponse<WeatherDetailResponseDto> getTodayWeather(@PathVariable("id") Long locationId) {
```

Comment:
`@RestController` 는 저희가 가공한 데이터를 Json 형태로 객체 데이터를 반환하여 HTTP Response Body에 담아 반환하는 역할을 수행합니다. 이를 위해서는 아래와 같이 `ResponseEntity`로 데이터를 감싸 반환할 필요가 있을 것 같습니다. 제가 기존에 작성한 LocationController를 참고해주세요!
```suggestion
    public ResponseEntity<ApiResponse<WeatherDetailResponseDto>> getTodayWeather(@PathVariable("id") Long locationId) {
```

Reply:
아래의 다른 API도 동일한 형식으로 데이터 구조를 변경해주시면 될 것 같습니다.